<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlightAssistant</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="gyro_1">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>400</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graph">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1321</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>1221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="graphic">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>30</y>
      <width>481</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="vid1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>900</width>
      <height>522</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="vid1_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>460</y>
      <width>900</width>
      <height>522</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="startpoint_vid2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>990</y>
      <width>831</width>
      <height>40</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="startpoint_vid1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>990</y>
      <width>831</width>
      <height>40</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="play_vid1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>992</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_vid2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>990</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="hight">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gyro_2">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>400</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gyro_3">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>400</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="vel">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>60</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <zorder>graphic</zorder>
   <zorder>gyro_1</zorder>
   <zorder>graph</zorder>
   <zorder>slider</zorder>
   <zorder>vid1</zorder>
   <zorder>vid1_2</zorder>
   <zorder>startpoint_vid2</zorder>
   <zorder>startpoint_vid1</zorder>
   <zorder>play_vid1</zorder>
   <zorder>play_vid2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>hight</zorder>
   <zorder>gyro_2</zorder>
   <zorder>gyro_3</zorder>
   <zorder>vel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
